utils/div_ceil.hh

```diff
@@ -9,8 +9,18 @@
 #pragma once
 
 #include <concepts>
+#include <limits>
+#include "utils/assert.hh"
 
+template <std::integral Divdend, std::integral Divisor>
```

Comment:
Typo: `Divdend` -> `Divident`

Reply:
good catch (although your correction also contains a typo, just a different one :) )

Reply:
Whoops, you're right :')